README.md

```diff
@@ -1,45 +1,61 @@
-2. The simulated data is then reconstructed using each of the reconstruction algorithms. At the moment, these algorithms are: 
-- [Gridrec](/docs/gridrec.md)
-- [MLEM](/docs/mlem.md)
-- [OSEM](/docs/osem.md)
-- [OSPML Hybrid](/docs/osmpl_hybrid.md)
-- [OSMPL Quad](/docs/ospml_quad.md)
-- [PML Hybrid](/docs/pml_hybrid.md)
-- [SIRT](/docs/sirt.md)
-- [SIRT_GPU](/docs/sirt_gpu.md)
 
-To view the documentation for these functions, go to the TomoPy [readthedocs](https://tomopy.readthedocs.io/en/latest/api/tomopy.recon.algorithm.html) website.
+### CPU Summary
+The following graph shows how the MS-SSIM index improves over-time for the CPU algorithms. 
 
-3. Each of the reconstructed images is then measured against the original image to quantify reconstruction quality. The quality of the reconstructions is plotted as a function of time and saved.
+![Image](/2021-02-19/cpu/peppers/summary.svg)
 
-### Limitations
+### GPU Summary
+The following graph shows how the MS-SSIM index improves over-time for the GPU algorithms. 
+
+![Image](/2021-02-19/gpu/peppers/summary.svg)
+
+Below are the "best" images for each of the available reconstruction algorithms.
+
+### Gridrec
+![Image](/2021-02-19/cpu/peppers/gridrec-0.png)
+### MLEM
+![Image](/2021-02-19/cpu/peppers/mlem-5.png)
+### OSEM
+![Image](/2021-02-19/cpu/peppers/osem-5.png)
+### OSPML Hybrid
+![Image](/2021-02-19/cpu/peppers/ospml_hybrid-5.png)
+### OSPML Quad
+![Image](/2021-02-19/cpu/peppers/ospml_quad-5.png)
+### PML Hybrid
+![Image](/2021-02-19/cpu/peppers/pml_hybrid-5.png)
+### SIRT 
+![Image](/2021-02-19/cpu/peppers/sirt-5.png)
+### SIRT GPU
+![Image](/2021-02-19/gpu/peppers/sirt_cuda-17.png)
+
+## Limitations
 
 While they are able to account for noise, these benchmarks currently do not account for the possibility of several classes of artifacts that may occur in realistic conditions. These include
 - Sample-based artifacts: 
-  - Motion artifacts: If part of the sample is moving during data collection, there can be blurriness in the reconstruction.
+  - Motion artifacts: If part of the sample moves during data collection, there can be blurriness in the reconstruction.
   - Out of field artifact: Dark streaks can appear when part of the sample is outside the field-of-view of the detector.
 - Physics-based artifacts
   - Beam hardening: If an X-ray is polychromatic, materials with a high atomic number can absorb the low-energy X-rays more than the high-energy X-rays.
   - Photon starvation: When there are regions of relatively high attenuation in a sample, dark streaks can form as a result of few photons reaching the detector. 
 - Hardware-based artifacts:
-  - Ring artifact: If a detector is improperly calibrated of otherwise faulty, a faint ring will appear in the reconstruction.
+  - Ring artifact: If a detector is improperly calibrated or otherwise faulty, a faint ring will appear in the reconstruction.
+
```

Comment:
Include a bulleted list of artifacts that are included. Having two explicit lists: included artifacts and excluded artifacts is clearer.